src/worldofzuul/Game.java

```diff
@@ -4,6 +4,11 @@
  * @author  Michael Kolling and David J. Barnes
  * @version 2006.03.30
  */
+
+/**
+ * 
+ * @author ander
+ */
```

Comment:
Denne del må gerne fjernes eller i hvertfald færdigøres. 😄

Reply:
Fixed in: 0654746aa05d460cce2b922ea2b2f5f9fad6f4f6